test/parallel/test-internal-util-toInteger.js

```diff
@@ -0,0 +1,19 @@
+// Flags: --expose-internals
```

Comment:
This should probably test for a wider range of values. `test/parallel/test-net-internal.js` has some good examples.